.github/workflows/publish.yml

```diff
@@ -4,10 +4,17 @@ on:
   push:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+*" # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
-
+permissions:
+      packages: write
+      contents: read
 jobs:
-  publish-sdk-java:
+  publish-docker:
+    uses: ./.github/workflows/publish-docker.yaml
```

Comment:
are we going to publish even if the tests fails?

Reply:
It runs server tests inside of it